ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/sql/SqlLedger.scala

```diff
@@ -88,21 +88,27 @@ private[sandbox] object SqlLedger {
 
     override def acquire()(implicit context: ResourceContext): Resource[Ledger] =
       for {
-        _ <- Resource.fromFuture(new FlywayMigrations(jdbcUrl).migrate())
         dao <- ledgerDaoOwner(servicesExecutionContext).acquire()
-        _ <- startMode match {
-          case SqlStartMode.AlwaysReset =>
-            Resource.fromFuture(dao.reset())
-          case SqlStartMode.ContinueIfExists =>
-            Resource.unit
-        }
         retrievedLedgerId <- Resource.fromFuture(dao.lookupLedgerId())
         ledgerId <- Resource.fromFuture(retrievedLedgerId.fold(initialize(dao))(resume))
         _ <- Resource.fromFuture(initOrCheckParticipantId(dao, Tag.unwrap(participantId)))
         ledgerEnd <- Resource.fromFuture(dao.lookupLedgerEnd())
         ledgerConfig <- Resource.fromFuture(dao.lookupLedgerConfiguration())
         dispatcher <- dispatcherOwner(ledgerEnd).acquire()
         persistenceQueue <- new PersistenceQueueOwner(dispatcher).acquire()
+        _ <- startMode match {
```

Comment:
Why has this moved down?

How are all those DAO lookups above gonna work if the migrations haven't run yet?

Reply:
You need to move this back up to get the tests to pass.